bftengine/include/bftengine/SimpleClient.hpp

```diff
@@ -21,6 +21,7 @@
 #include "Metrics.hpp"
 #include "communication/ICommunication.hpp"
 #include "PerformanceManager.hpp"
+#include "../../../bftclient/include/bftclient/base_types.h"
```

Comment:
Ideally, we can `#include "bftclient/base_types.h`. If this doesn't work because the bftclient CMake target doesn't set its include dir properly then don't worry. We fix it later.

Reply:
NP